jupyter-docker/construct-spark-defaults.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+cd /etc
+
+# /etc/spark/conf/spark-defaults.conf has been mounted in during docker-compose
+# $HAILHASH has been ENV'd in from the Dockerfile
+# spark-defaults.conf.template is ADDed in the Dockerfile
+
+echo "Running construct-spark-defaults.sh:" >> /etc/construct-spark-defaults.log
+echo "HAILHASH: $HAILHASH" >> /etc/construct-spark-defaults.log
+echo "SPARK_HOME: $SPARK_HOME" >> /etc/construct-spark-defaults.log
+
+# Move the mounted spark-defaults.conf to the right place
+cp /etc/spark-defaults.conf $SPARK_HOME/conf/spark-defaults.conf
```

Comment:
note that `$SPARK_HOME` in the container is not `/etc/spark/conf` like it is on the outside. Instead it's something weird like `/etc/spark-version/`